Add hamburger menu and navmenu

diarisdiakite opened the pull request:
In this PR, I:
- Fix accessibility and performance issues by adding high-resolution images and the overall structure of the code
- Create the hamburger menu bars with _span_
- Added the hamburger nav menu (pair programming)
- Added the script to toggle the hamburger menu (pair programming)
- Added the script to open and close the hamburger nav menu (pair programming)
- Added javascript linters
- Fixed Javascript and Css linters errors...

mukhammadrizooff reviewed (commented):
Hi @diarisdiakite, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't pass your linter errors, kindly fix your linters 👍🏼 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

AbelG101 commented on assets/css/styles.css:
```diff
@@ -630,29 +660,115 @@ header .display-mobile img {
   }
 
-  header .display-mobile {
+  .header-nav .display-mobile {
     width: 100%;
     display: flex;
     justify-content: flex-end;
     align-items: flex-end;
   }
```
- [ ] Good Job 👏 so far but you still need to improve on some things, I noticed a discrepancy between the side menu in your design and the design in the Figma mockup. I understand that it can be tempting to deviate from the design when building a project; however, I would recommend that you adhere to the design as closely as possible to ensure a professional and polished final product.
     - Figma design:
       ![firefox_2mMVkGRmce](https://user-images.githubusercontent.com/104968236/231766464-74203787-42ec-43df-8a7b-e0e7d7ec4072.png)
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/231766677-3b6b9bcf-628b-4946-9b4a-01f284c45ba3.png)
- [ ] Please make sure you follow the proper background color, font color, font size, and Icons for the cross as specified in the design
- [ ] Also kindly fix the height on your design as it should almost fill the screen with a small space in the bottom Kindly check the design again.

diarisdiakite replied:
Done @AbelG101

AbelG101 commented on package.json:
### THIS COMMENT IS REGARDING THE README OF YOUR PROJECT AND NOT THE PACKAGE.JSON
- [ ] Please make sure your readme is always looking professional and functional as I noticed the table of contents section is not working as it should. The texts in the table of contents should be a link as they're used to easily navigate to the different sections by the user I noticed you changed them to headings so please make sure to fix them as they're the first thing any visitor is going to look at before exploring your project.
- Current state of your readme:
  ![image](https://user-images.githubusercontent.com/104968236/231771187-d50737c1-3531-493a-a5fe-68b82a79e2c2.png)

diarisdiakite replied:
Done @AbelG101

AbelG101 reviewed (changes requested):
![](https://media.giphy.com/media/ReyD4OdRiHeLDCRaEP/giphy.gif)


# Hi @diarisdiakite 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- You just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

diarisdiakite commented:
In this milestone, I:
- changed the background color of the hamburger navmenu
- Styled the links text according to the figma template
- Added the links to the readme sections
-

IngDavidTM reviewed (changes requested):
## Hi @diarisdiakite 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media2.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>

## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [ ] Your styling of the popup menu is exceptional :+1:, but as the previous reviewer requested, it is necessary to ensure that the menu occupies the entire screen size, as demonstrated in the Figma template. This is important because it will provide a consistent user experience and make the menu more accessible and user-friendly. Please make the necessary adjustments to ensure that the menu is appropriately sized.

**Your design**

![Screenshot from 2023-04-13 20-49-36](https://user-images.githubusercontent.com/105117832/231921099-f442360b-d801-4ca8-a8cb-70a78655a9e0.png)

**Figma**

![Screenshot from 2023-04-13 20-50-37](https://user-images.githubusercontent.com/105117832/231921124-e645d353-4a45-4385-a8c5-c860ad68f40d.png)


- [x] _Check the comments under the review._ ⬇

## Optional suggestions

- [OPTIONAL] I think it might be more beneficial to utilize the `position: fixed` property in the mobile menu to restrict user access to the rest of the page. The reason behind this recommendation is to maintain a more focused user experience, preventing them from getting lost in the content of the page and allowing them to concentrate solely on the menu options. It can also help make the design more consistent and user-friendly across different screen sizes, ensuring that the menu remains accessible and easy to use :+1:.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
